moveit_simple/include/moveit_simple/online_robot.h

```diff
@@ -60,25 +60,30 @@ class OnlineRobot : public Robot
    * prior calls to "addTrajPoint".
    * @param collision_check - bool to turn check for collision on\off
    * @param retime_trajectory - bool to fix trajectory timesteps if necessary on\off
+   * @param max_velocity_scaling_factor - double for limiting the joint velocities if you retime trajectory
+   * @param max_acceleration_scaling_factor - double for limiting the joint velocities if you retime trajectory
    * @throws <moveit_simple::ExecutionFailureException> (Execution failure)
    * @throws <moveit_simple::IKFailException> (Conversion to joint trajectory failed)
    * @throws <std::invalid_argument> (Trajectory "traj_name" not found)
    * @throws <moveit_simple::CollisionDetected> (One of interpolated point is
    * in Collision with itself or environment)
    */
-  void execute(const std::string traj_name, bool collision_check = false, bool retime_trajectory = false);
+  void execute(const std::string traj_name, bool collision_check = false, bool retime_trajectory = false,
+               double max_velocity_scaling_factor = 1.0, double max_acceleration_scaling_factor = 1.0);
```

Comment:
I'm not sure we should add this to the execute() function since these parameters are only used in rare cases (retime_trajectory=true and trajectory is invalid). I would prefer just adding a getter/setter for these parameters.